frontend/e-buddy/src/app/(demo)/bots/setup/page.tsx

```
@@ -1,153 +1,251 @@
+                  name="faqInfo"
+                  value={formData.faqInfo}
+                  onChange={handleChange}
+                  placeholder="Add company details, FAQs, or any information your bot should know"
+                  className="w-full border border-gray-200 rounded-lg px-4 py-3 shadow-sm form-input-focus resize-none"
+                  rows={3}
+                ></textarea>
+              </div>
+              <div>
+                <label className="block text-gray-700 font-medium mb-2">
+                  Customer Support Contact
+                </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)